flpsuitedevenv.sh

```diff
@@ -0,0 +1,32 @@
+package: FLPSuiteDevEnv
+version: "1.0.0"
+tag: "FLPSuiteDevEnv-1.0.0"
+requires:
+  - CMake
+  - O2-customization
+  - alibuild-recipe-tools
+  - ms_gsl
+  - RapidJSON
+  - GMP
+  - MPFR
+  - cub
```

Comment:
cub has been removed from alidist, I think it should be removed here

Reply:
Indeed, thank you